sdk-lib/src/main/java/cash/z/ecc/android/sdk/model/LightWalletEndpointExt.kt

```diff
@@ -31,7 +31,7 @@ val LightWalletEndpoint.Companion.Mainnet
 
 val LightWalletEndpoint.Companion.Testnet
     get() = LightWalletEndpoint(
-        "testnet.lightwalletd.com",
+        "lightwalletd.testnet.electriccoin.co",
```

Comment:
Should we keep this change?

I was unable to test without making this change but it will be incorporated into all SDK clients.

Reply:
I think it's fine.